App.js

```diff
@@ -1,17 +1,77 @@
-import * as React from 'react'
-import { WebView } from 'react-native-webview'
+import React, { useRef } from "react";
+import { WebView } from "react-native-webview";
+
+// Replace with your custom subdomain
+const subdomain = "demo";
+
+let isSubscribed = false;
+let count = 0;
+const correlationId = "a0bf9c2a-44d7-4882-8e72-4bc7ab73849f";
+
+function OnAvatarExported(json) {
+  alert(`Avatar Url = ${json.data?.url}`);
+}
+
+const View = () => {
+  const webview = useRef();
+  const subscribe = () => {
+    if (isSubscribed) {
```

Comment:
I'd suggest using `useEffect` to subscribe. Would be no need to use a boolean to track this.

Reply:
hmm I tried this but it didn't seem to work? 

```js
const View = () => {
  const webview = useRef();

  useEffect(() => {
    webview.current.postMessage(
      JSON.stringify({
        target: "readyplayerme",
        type: "subscribe",
        eventName: "v1.avatar.exported",
      })
    );
  });
  ....
```